src/MaintenanceModeCommand.php

```diff
@@ -132,7 +132,22 @@ private function get_maintenance_mode_status() {
 
 		$maintenance_file = trailingslashit( $wp_filesystem->abspath() ) . '.maintenance';
 
-		return $wp_filesystem->exists( $maintenance_file );
+		if ( ! $wp_filesystem->exists( $maintenance_file ) ) {
+			return false;
+		}
+
+		require ABSPATH . '.maintenance';
```

Comment:
here the thing is a bit complex since the `$upgrading`  can be either a timestamp value or a result of an expression such as `time()`. 
I've added here the regex which I wanted to use https://regex101.com/r/ayl6x5/1 but I feel like is not any better than the `require` version since it still allows running anything there for `$upgrading` variable. 

Some idea is to limit the regex to the `time()` function or `timestamp` format.

Reply:
> here the thing is a bit complex since the `$upgrading` can be either a timestamp value or a result of an expression such as `time()`.

Can it? WordPress only puts the timestamp in there. Which tool does something else?

Reply:
I've seen this too. While WordPress only puts the timestamp in, plugins, system operators, etc. can put much more custom logic in it.

We could:
1. Process `$upgrading` correctly when it's only a timestamp.
2. Output `WP_CLI::warning()` when we detect `time()` and leave the current behavior.